models/migrations/v1_18/v230_test.go

```diff
@@ -36,7 +36,7 @@ func Test_AddConfidentialClientColumnToOAuth2ApplicationTable(t *testing.T) {
 	}
 
 	got := []ExpectedOAuth2Application{}
-	if err := x.Table("o_auth2_application").Select("id, confidential_client").Find(&got); !assert.NoError(t, err) {
+	if err := x.Table("oauth2_application").Select("id, confidential_client").Find(&got); !assert.NoError(t, err) {
```

Comment:
is a new migration needed to rename the table if it is incorrectly named?

Reply:
Now the database should have both tables. The `oauth2_application` is the correct one and `o_auth2_application` is the incorrect.

Reply:
Maybe migration to drop the incorrect table?

Reply:
f8d57dc